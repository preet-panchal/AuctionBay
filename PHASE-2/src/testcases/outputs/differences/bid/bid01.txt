--- ./../testcases/outputs/expected/bid/bid01.txt	2023-02-03 02:04:49
+++ ./../testcases/outputs/actual/bid/bid01.txt	2023-03-05 16:32:33
@@ -13,10 +13,9 @@
 
 Enter operation:
 Enter username:
-Hello ssuser01 (SS)!
 
+Error: This user does not exist in the user-accounts file.
 Enter operation:
-Error: Only AA, FS, or BS have privileges for this operation.
+Error: You must login first.
 
 Enter operation:
-ssuser01 successfully logged out.
\ No newline at end of file
