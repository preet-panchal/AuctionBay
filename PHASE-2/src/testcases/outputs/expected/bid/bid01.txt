Welcome to AuctionBay!
--
Here is a list of operations (what to input):
- Login (login)
- Create (create)
- Delete (delete)
- Advertise (advertise)
- Bid (bid)
- Refund (refund)
- Add Credit (addcredit)
- List All Items (listall)
- Logout (logout)

Enter operation:
Enter username:
Hello User04 (SS)!

Enter operation:
Error: Only AA, FS, or BS have privileges for this operation.

Enter operation:
User04 successfully logged out.
